core/store/src/lib.rs

```diff
@@ -295,8 +300,8 @@ pub fn read_with_cache<'a, T: BorshDeserialize + 'a>(
     Ok(None)
 }
 
-pub fn create_store(path: &Path) -> Store {
-    let db = Arc::new(RocksDB::new(path).expect("Failed to open the database"));
+pub fn create_store(path: &Path, enable_statistics: bool) -> Store {
```

Comment:
Adding an `false` to the dozens of call sites doesn’t feel great. Perhaps we need api shaped like this: 

```
/// Creates store with default config.
/// Use this if you don’t know what you are doing.
pub fn create_store(path: &Path)

/// Opens store with extra control knobs
pub fn create_store_conf(path: &Path, confit: StoreConfig)

#[derive(Default)]
pub struct StoreConfig {
  pub read_only: bool,
  pub enable_stats: bool,
}

Reply:
Done, this is indeed cleaner